xAPI-About.md

```diff
@@ -522,12 +518,7 @@ specification requires that all Statements have a "verb" property, but does not
 can be used. This flexibility enables xAPI to be used in any context, including future use cases not envisaged by the 
 specification authors.
 
-It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, 
-contextual relationships, extensions, etc. 
-to be used in their profiles by providing unique identifiers wherever applicable. The CoP will define these identifiers and 
-their metadata in a [Vocabulary](#def-vocabulary). A profile is a set of rules and vocabularies to implemented in addition to 
-xAPI for the particular use case being addressed. It is very important that such communities exist and share best practices. 
-For more information on publishing vocabularies, see the [Vocabulary Companion Specification](https://adl.gitbooks.io/companion-specification-for-xapi-vocabularies/content) and [Vocabulary Primer](https://adl.gitbooks.io/experience-xapi-vocabulary-primer/content/) documents.
+It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, contextual relationships, extensions, etc. to be used in their profiles by providing unique identifiers identifiers and their metadata in a [Profile](#def-profile). A profile is simply the documentation of application-specific vocabulary, extensions, and statement templates for use case being addressed. It is very important that such communities exist and share best practices. For more information on publishing profiles, see the [Profile Companion Specification](https://github.com/adlnet/xapi-profiles).
```

Comment:
This is a very long line which makes it difficult to review changes. Could this be broken up into similar lines to the orginal so its clearer to see whats changed?

Reply:
Most replaced "vocab" links and references with new profile guidance. It's only three sentences now. What do you want me to do here @garemoko? I don't think this really lends itself to bullets like the previous version did.

Reply:
Its about the underlying markdown rather than whats displayed. The original markdown had this section split across multiple lines. If this change didn't change the line breaks it would be easier to see what exactly has changed. Also, if this line was broken up into multiple lines it will be able to review future changes to this text for similar reasons. 

If you look at the rest of the spec you'll see roughly how long all the lines are. Single line breaks in the markdown don't affect how the text is displayed at all.

Reply:
Gotcha. Thanks man. Appreciate the clarification. I'll work on resubmitting this as soon as I can.